### DISCUSSION TITLE
Wildcard domain in url field

### DISCUSSION THREAD

**Discussion by LickABrick** [pocketbase/pocketbase#3532] [Q&A] [answered]
In my app I have the need to save a url to PocketBase. This url is generated by a 3rd party (microsoft in this case). The url looks like this: <365 tenant name>.webhook.office.com. Because the first part of the domain is different for every 365 tenant I would like to use a wildcard allowed domains like *.webhook.office.com. Currently this is not possible.

Would this be a valueable addition to the allowed domains filter?

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure if this would be necessary.

One possible workaround could be to to use a basic `text` field and define a Go regex pattern that will satisfy your url criteria.

**Comment by LickABrick** [comment]
That would also be a valid option. Thanks didnt think of that.
